runtime/near-runtime-fees/src/lib.rs

```diff
@@ -323,11 +363,17 @@ impl RuntimeFeesConfig {
                 deploy_cost_per_byte: 0,
                 funcall_cost_base: 0,
                 funcall_cost_per_evm_gas: 0,
-                ecrecover_cost: 0,
-                sha256_cost: 0,
-                ripemd160_cost: 0,
-                identity_cost: 0,
-                modexp_cost: 0,
+                precompile_costs: EvmPrecompileCostConfig {
+                    ecrecover_cost: EvmLinearCost { base: 0, word: 0 },
+                    sha256_cost: EvmLinearCost { base: 0, word: 0 },
+                    ripemd160_cost: EvmLinearCost { base: 0, word: 0 },
+                    identity_cost: EvmLinearCost { base: 0, word: 0 },
+                    modexp_cost: EvmModexpCost { divisor: 0 },
```

Comment:
Can a divisor be `0`? Maybe it should be at least `1`, but then we need to handle free case.

I suggest we implement that if `divisor == 0` then gas `0` in `ModexpPricer`

Reply:
Good point